src/Icons/PfIcons/add-circle-o-icon.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Might better to only check-in the Json file and just have these generated at build. That way the Json file doesn't get out of sync.

Reply:
Mhhh, yeah that makes sence, I'll work on it.

Reply:
Filed https://github.com/patternfly/patternfly-next/issues/771. Let's revisit this later, would like to unblock the PR if possible!